### PR TITLE
feat: Devnet stop backward fill

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
changes:
- indexer backward filling only at startup and until the oldest tx is found, then stop to reduce rpc calls
- fix docker deploy process
- debugged hosted relayer docker instance

### CODE REVIEW

File: scripts/buildDockerRelayer.sh [hunk 1]
```diff
@@ -51,12 +49,19 @@ cleanup() {
...
 
+    echo "Deleting NPM artifacts..."
+    rm -rf $(dirname $0)/../zk.js/package-lock.json
+    rm -rf $(dirname $0)/../relayer/package-lock.json
+    rm -f $(dirname $0)/../pnpm-lock.yaml
+
+
     echo "Rebuilding workspace..."
-    $(dirname $0)/build.sh
+    # bash $(dirname $0)/build.sh
```
Comment (ananas-block) [inline scripts/buildDockerRelayer.sh:64]: is this commented on purpose?

**Review by ananas-block — COMMENTED** [review]
Couple questions, and please let's not add key material to the repo.

File: scripts/buildDockerRelayer.sh (hunk 1 above)
Comment (SwenSchaeferjohann) [reply scripts/buildDockerRelayer.sh]: fixed thx

File: Dockerfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ WORKDIR /usr/src/app
 COPY ./relayer /usr/src/app/relayer
 
 COPY ./docker-entrypoint.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/docker-entrypoint.sh
```
[the remark this answers is not in the document]
Comment (SwenSchaeferjohann) [reply Dockerfile]: nope not necessary, removed

**Review by ananas-block — APPROVED** [review]

**Pull request merged**